spec.html

```diff
@@ -43383,6 +43414,44 @@ <h1>The Strict Mode of ECMAScript</h1>
   </ul>
 </emu-annex>
 
+<emu-annex id="sec-host-layering-points">
+  <h1>Host Layering Points</h1>
+  <p>See <emu-xref href="#sec-hosts-and-implementations"></emu-xref> for the definition o host.</p>
+  <emu-annex id="sec-host-hooks-summary">
+    <h1>Host Hooks</h1>
+    <p><b>HostEnqueuePromiseJob(...)</b>: See <emu-xref href="#sec-hostenqueuepromisejob"></emu-xref></p>
+    <p><b>HostEnsureCanCompileStrings(...)</b>: See <emu-xref href="#sec-hostensurecancompilestrings"></emu-xref></p>
+    <p><b>HostFinalizeImportMeta(...)</b>: See <emu-xref href="#sec-hostfinalizeimportmeta"></emu-xref></p>
+    <p><b>HostGetImportMetaProperties(...)</b>: See <emu-xref href="#sec-hostgetimportmetaproperties"></emu-xref></p>
+    <p><b>HostHasSourceTextAvailable(...)</b>: See <emu-xref href="#sec-hosthassourcetextavailable"></emu-xref></p>
+    <p><b>HostImportModuleDynamically(...)</b>: See <emu-xref href="#sec-hostimportmoduledynamically"></emu-xref></p>
+    <p><b>HostPromiseRejectionTracker(...)</b>: See <emu-xref href="#sec-host-promise-rejection-tracker"></emu-xref></p>
+    <p><b>HostResolveImportedModule(...)</b>: See <emu-xref href="#sec-hostresolveimportedmodule"></emu-xref></p>
+    <p><b>InitializeHostDefinedRealm(...)</b>: See <emu-xref href="#sec-initializehostdefinedrealm"></emu-xref></p>
```

Comment:
InitializeHostDefinedRealm is different than the others, in that hosts don't fully define it. Maybe it'd be simpler if they did though...

Reply:
@annevk expressed desire to not have it be so awkward to use. I'll tackle that after this PR.